script/Base.s.sol

```diff
@@ -15,6 +15,9 @@ contract BaseScript is Script {
     /// @dev The default value for `segmentCountMap` and `trancheCountMap`.
     uint256 internal constant DEFAULT_MAX_COUNT = 500;
 
+    /// @dev The address of the Sablier deployer.
+    address internal constant SABLIER_DEPLOYER = 0xb1bEF51ebCA01EB12001a639bDBbFF6eEcA12B9F;
```

Comment:
Will it always be same?

Reply:
I’m not sure what you mean.

If there is a multisig set in the `adminMap`, we will use it. Otherwise, we use the default admin, which is the Sablier deployer EOA

Reply:
Closing this thread. Nor relevant anymore.